.admin/public_html/arduboy/template.html

```diff
@@ -47,12 +83,16 @@ <h1>{{repository}}</h1>
 					<!-- Boxes -->
 						<div class="thumbnails">
 {{#items}}
-							<div class="box" data-item-tags="{{genre}}" data-item-date="{{date}}">
+							<div class="box" data-item-tags="{{genre}}" data-item-date="{{date}}" data-item-license="{{license}}">
 								<a class="image fit"><img src="{{banner}}" alt="" /></a>
 								<div class="inner">
 									<h3>{{title}} by <a href="{{sourceUrl}}" title="View source code" target="_blank" >{{author}}</a></h3>
 									<p>{{description}}</p>
+									{{#license}}
```

Comment:
[20-Feb-2018 17:04:21 America/Chicago] PHP Fatal error:  Uncaught exception 'Mustache_Exception_SyntaxException' with message 'Nesting error: license (on line 90) vs. items (on line 103)' in /home/erwinrie/public_html/arduboytest/Mustache/Parser.php:126
Stack trace:
#0 /home/erwinrie/public_html/arduboytest/Mustache/Parser.php(105): Mustache_Parser->buildTree(Array, Array)
#1 /home/erwinrie/public_html/arduboytest/Mustache/Parser.php(105): Mustache_Parser->buildTree(Array, Array)
#2 /home/erwinrie/public_html/arduboytest/Mustache/Parser.php(43): Mustache_Parser->buildTree(Array)
#3 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(745): Mustache_Parser->parse(Array)
#4 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(759): Mustache_Engine->parse('<!DOCTYPE HTML>...')
#5 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(700): Mustache_Engine->compile('<!DOCTYPE HTML>...')
#6 /home/erwinrie/public_html/arduboytest/Mustache/Engine.php(619): Mustache_Engine->loadSource('<!DOCTYPE HTML>...')
#7 /home/erwinrie/public_html/arduboytest/Mustache/ in /home/erwinrie/public_html/arduboytest/Mustache/Parser.php on line 126